services/frontend/Dockerfile

```diff
@@ -28,8 +28,11 @@ RUN yarn install --ignore-optional --frozen-lockfile --network-timeout 1000000 -
 # Copy app source
```

Comment:
Hey @kuronosec, I put together a quick multi-stage build for docker
https://gist.github.com/danazkari/ae9d9aab2c913a73f451f7bacdbd8400
There are a couple of files that we need to place in the frontend service directory (nginx.conf and .dockerignore), so that instead of having a heavy node server running, we have a lightweight nginx hosting the built services.

Let me know what you think about it and if you have questions about how it works, just make sure that you add the `target` parameter to point at `server` stage.

Reply:
@danazkari ok thanks! I'll use. One question I have now is what to do with the CI container inside the frontend service. I just rename it to Dockerfile.ci, don't know if that will break something.

Reply:
Well now that you mention it, I don't think we're using that anymore, since that was for netifly and I think we had already sacked that account, right @xavier506 ? 

And I think the unspoken "standard" is to name it something like `<context>.Dockerfile` so we get editor syntax goodies!

Reply:
We are no longer using netlify.